test/specs/config/generator.js

```diff
@@ -5,11 +5,13 @@ var fs = require('fs');
 var Vow = require('vow');
 
 var Checker = require('../../../lib/checker');
+var Configuration = require('../../../lib/config/configuration');
 var ConfigGenerator = require('../../../lib/config/generator');
+var crockfordClone = require('../../data/configs/generator/crockfordClone');
```

Comment:
Although you would need only couple rules from that preset, i'd say it doesn't really matter, we just shouldn't forget we would need to update that when we rename the rules